pkg/cryptoEAS/index.go

```
@@ -0,0 +1,37 @@
+
+func encryptAES(plaintext string, key []byte) ([]byte, []byte) {
+	block, _ := aes.NewCipher(key)
+	nonce := make([]byte, 12)
+	io.ReadFull(rand.Reader, nonce)
+
+	aesgcm, _ := cipher.NewGCM(block)
+	ciphertext := aesgcm.Seal(nil, nonce, []byte(plaintext), nil)
+	return ciphertext, nonce
+}
+
+func decryptAES(ciphertext, key, nonce []byte) string {
```

golangci-lint (1.64.8)

func `decryptAES` is unused

(unused)